nullaway/src/test/java/com/uber/nullaway/NullAwayAssertionLibsTests.java

```diff
@@ -365,6 +365,67 @@ public void supportAssertJAssertThatIsNotNull_Object() {
         .doTest();
   }
 
+  @Test
+  public void supportAssertJAssertThatIsNotNullWithDescription_Object() {
+    makeTestHelperWithArgs(
+            Arrays.asList(
+                "-d",
+                temporaryFolder.getRoot().getAbsolutePath(),
+                "-XepOpt:NullAway:AnnotatedPackages=com.uber",
+                "-XepOpt:NullAway:HandleTestAssertionLibraries=true"))
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import java.lang.Object;",
+            "import java.util.Objects;",
+            "import javax.annotation.Nullable;",
+            "import static org.assertj.core.api.Assertions.assertThat;",
+            "class Test {",
+            "  private void foo(@Nullable Object o) {",
+            "    assertThat(o).as(\"test\").isNotNull();",
```

Comment:
question: can we chain multiple `as()`? i.e., `assertThat(o).as("test1").describedAs("test2").isNotNull()`?

If so we could probably add this case since this PR handles that as well.

Reply:
Yup this works, test in d9ec5f33f3d4021a32b82535e7067da8996266a7